src/strategies/bravearena/index.ts

```diff
@@ -0,0 +1,58 @@
+import fetch from 'cross-fetch';
+// const { getAddress } = require('@ethersproject/address');
+
+export const author = 'bravearena';
```

Comment:
PR should be coming from this github's fork

Reply:
yes